concepts/numbers/introduction.md

```diff
@@ -0,0 +1,53 @@
+# Introduction
+The built-in number types in C++ can be divided into integers and floating points.
+Integers are whole numbers like `0`, `691`, or `-2`.
+Examples of floating point numbers are `6.02214076`, `0.1`, or `-1.616`.
+
+## Integers
+When you assign a value to an `int` variable, you can do so directly with a literal.
+There are different integer literals based on the base of the representation.
+Decimal integer literals are the most common and use the numbers `0` to `9`.
+By adding a special prefix it is possible to use other bases.
+The example below shows the number `9421` in its four representations and prefixes.
+
```

Comment:
With my Beginner hat on, I have now finished the above paragraph but am not much wiser. I think examples first would be better: then it would be clearer what the words (literal, decimal literal, prefixes, bases, representations) refer to.